.github/ISSUE_TEMPLATE/support-onboard-a-new-product.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Support: Onboard a new product'
```

Comment:
Suggestion for clarification:
- "Onboard product team on consortium demo environments"

Reply:
adapted your proposals to make it clear where we create things, see https://github.com/eclipse-tractusx/sig-infra/pull/10/commits/54e87d6f9a8ee097475f50af445802f520002e39